explorer/explorer.go

```diff
@@ -293,6 +294,20 @@ func New(dataSource explorerDataSource, userRealIP bool) *explorerUI {
 			t, _ := time.Now().Zone()
 			return t
 		},
+		"fancyDCR": func(v float64) []string {
+			roundedV := fmt.Sprintf("%.8f", v)
+			trailingZeroes := ""
+			for i := range roundedV {
+				if roundedV[len(roundedV)-1-i] != 48 {
+					break
+				}
+				trailingZeroes += "0"
```

Comment:
This seems OK, but in general I try to use the standard library where possible.  In this case, `roundedV = strings.TrimRight(roundedV, "0")` would get you most of the way, and then you just need `trailingZeros := string.Repeat("0", oldLength - newLength)`

Reply:
Well aren't those handy! That's much better.
Pushed another commit using strings lib.